### PR TITLE
Migrate to github actions, qualify package names

### PR DESCRIPTION
**Pull request by r-ash**
This is similar to eppasm PR https://github.com/mrc-ide/eppasm/pull/42 

I'm trying to build Naomi on github actions but I am seeing errors from first90release related to the use of unqualified functions. See e.g. https://github.com/mrc-ide/naomi/actions/runs/8285282918/job/22672679214#step:7:7673

I've moved this over to github actions whilst I am at it and tidied up some of the R CMD warnings and notes

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -122,7 +126,7 @@ Updates for 2022 UNAIDS estimates:
 
 # first90 1.4.3
 
-* Patch: in function `add_ss_indices()`, add argument `type.convert(..., as.is = TRUE)` 
+* Patch: in function `add_ss_indices()`, add argument `utils::type.convert(..., as.is = TRUE)`
```
Comment (jeffeaton) [inline NEWS.md]: don't think necessary to prefix in the NEWS entry

File: tests/testthat/Rplots.pdf
Comment (jeffeaton) [inline tests/testthat/Rplots.pdf:1]: is this file intentionally committed?

**Review by jeffeaton — COMMENTED** [review]
thanks -- question about one file whether intentionally committed.

File: tests/testthat/Rplots.pdf
Comment (r-ash) [reply tests/testthat/Rplots.pdf:1]: No, this is being generated when running the tests in RStudio. I've deleted it and added to .gitignore

File: NEWS.md (hunk 1 above)
Comment (r-ash) [reply NEWS.md]: Thanks, just overly enthusiastic search + replace. I've reverted that.

**Review by jeffeaton — APPROVED** [review]

**Pull request merged**